.github/workflows/main.yml

```diff
@@ -54,6 +54,7 @@ jobs:
       shell: bash -l {0}
       run: |
         pip install "traitlets>=4,<5"
+        pip install ipykernel==5.5.5
```

Comment:
Just to be sure we don't forget later, where is this issue being tracked?

Reply:
Sorry, I forgot about it, where should I put the issue?